certificate-policy.md

```diff
@@ -845,6 +857,8 @@ Individuals may only assume one of the Administrator, Officer, and Security role
 #### 5.3.1 Qualifications, experience, and clearance requirements
 All persons filling trusted roles shall be selected on the basis of loyalty, trustworthiness, and integrity, and shall be U.S. citizens.  The requirements governing the qualifications, selection and oversight of individuals who operate, manage, oversee, and audit the CA shall be set forth in the CPS.
 
+The FPKIPA shall be notified when a CA under this policy transfers ownership and/or management of the CA to a different organization by law, mandate, contract, or other means. Prior to ownership or management transfer, the new organization shall provide an audit in accordance with section 8.1 to demonstrate compliance with this CP. If the transfer includes a physical relocation of cryptographic modules, the transfer shall occur in accordance with section 5.1.1.
```

Comment:
"Organization" is where the logic may break down.  Organization is U.S. Government.

Reply:
I discussed this with a few on the team. It is tricky one in regards to government. This is in response to 8.1 (CA Ownership) in the Mozilla CP. The intent of this and 8.2 seem to speak to making sure the CA owner and CA operator (if they are separate) understand and can comply with the Mozilla CP. Understand being the bigger piece. If the government is the owner and not a specific agency, then this addition is not needed.